### ISSUE TITLE
Pascal VOC 2012 - training time

### ISSUE THREAD

**Issue by atiszabo** [hszhao/semseg#69]
Hi,

we are training Pascal VOC 2012 trainaug with PSPnet50. We have only 4 gpus, but the **training time is 10x more**, which is strange (we could even fit more into a batch).

**our setup:**
- 4 x P100 with 16GB
- batch size:20
- lr: 0.005

**Is DDP enabled by default?** **What else could cause the slowing down?** Only having 2x less GPUs is not proportionate with the speed drop. 

@zhixuanli you got close with cityscapes in  https://github.com/hszhao/semseg/issues/7#issuecomment-536251008 but still not the same. Did you only change the GPUs? Is that training time with distributed data parallel model?

@hszhao

**Comment by zhixuanli** [comment]
@atiszabo 

Hi, I haven't run any experiments on PASCAL VOC dataset and because it has been one year since then, so it's hard for me to give a clear and correct answer.

I remember that I only changed the GPU and turn on FP 16 precision with [Apex](https://github.com/NVIDIA/apex).

And maybe P100 is slower than 2080ti?

You can also try to enlarge the learning rate and watch the validation loss curve in case of an early stop. Usually bigger learning rate will cause faster convergence.

**Comment by zhixuanli** [comment]
@atiszabo 

Look at https://github.com/XiaLiPKU/EMANet for a better performance(88.2 on  PASCAL VOC test dataset).

**Comment by zhixuanli** [comment]
I suddenly remember that it will be faster if you turn off `cudnn` during training

**Comment by atiszabo** [comment]
Thank you for the suggestion.
Yes, using apex with fp16 could make the difference
